httpd/oidc.go

```diff
@@ -203,7 +203,19 @@ func (t *oidcToken) parseClaims(claims map[string]interface{}, usernameField, ro
 }
 
 func (t *oidcToken) isAdmin() bool {
-	return t.Role == "admin"
+	switch v := t.Role.(type) {
+	case string:
+		return v == "admin"
+	case []interface{}:
+		for _, s := range v {
+			if s.(string) == "admin" {
```

Comment:
Please change to something like this:

`if val, ok := s.(string); ok && val == "admin" {`

otherwise we'll get a panic if the slice contains something other than strings